### PR TITLE
Add new connector metadata namespace, change branch for metadata controller

### PR DESCRIPTION
**Pull request by Krenair**
Per request from Chris Wynne

### CODE REVIEW

File: clusters/sandbox-values.yaml [hunk 1]
```diff
@@ -5,11 +5,17 @@ global:
     ip: "10.101.29.134"
 
 namespaces:
+- name: sandbox-connector-node-metadata
+  owner: alphagov
+  repository: verify-metadata
+  branch: master
+  path: ci
```
Comment (samcrang) [inline clusters/sandbox-values.yaml]: This path doesn't appear to exist in [the repo](https://github.com/alphagov/verify-metadata) on `master`.

Comment (Wynndow) [reply clusters/sandbox-values.yaml]: No, that's the next thing I'm doing.

Comment (Wynndow) [inline clusters/sandbox-values.yaml]:
```suggestion
  path: ci/sandbox
```

File: clusters/sandbox-values.yaml [hunk 2]
```diff
@@ -5,11 +5,17 @@ global:
     ip: "10.101.29.134"
 
 namespaces:
+- name: sandbox-connector-node-metadata
```
Comment (blairboy362) [inline clusters/sandbox-values.yaml]: Alphabetical. Nice! :clap:

**Pull request merged**